node/src/accountant/mod.rs

```diff
@@ -697,6 +700,10 @@ impl Accountant {
             })
             .expect("UiGateway is dead");
     }
+
+    pub fn dao_factory(data_directory: &Path) -> DaoFactoryReal {
+        DaoFactoryReal::new(data_directory, false, MigratorConfig::panic_on_migration())
+    }
 }
 
 // At the time of this writing, Rust 1.44.0 was unpredictably producing
```

Comment:
Is there an issue we can watch to determine when the function below is no longer required?